pkg/operator/controller/route-status/controller_test.go

```diff
@@ -0,0 +1,197 @@
+package routestatus
+
+import (
+	"context"
+	v12 "github.com/openshift/api/operator/v1"
+	routev1 "github.com/openshift/api/route/v1"
+	"github.com/openshift/cluster-ingress-operator/pkg/operator/controller"
+	"github.com/openshift/cluster-ingress-operator/test/unit"
+	"github.com/stretchr/testify/assert"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"testing"
+)
+
+// Test_Reconcile verifies Reconcile
+// Note: This effectively tests clearStaleRouteAdmittedStatus as well
```

Comment:
Please use full sentences.  * grin *.  
```suggestion
// Test_Reconcile verifies Reconcile behaves as expected.
// Note: This effectively tests clearStaleRouteAdmittedStatus as well.
```